pandas.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""pandas.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1d-Bn6_9MT2YvqdNjhcWlrq3UWnIlWrCA
+"""
+
+import pandas as pd
+import matplotlib.pyplot as plt
+import numpy as np
+
+# функции read_gff и read_bed6 для чтения соответствующих форматов
+
+def read_gff(file):
+    col = ['chromosome', 'source', 'type', 'start', 'end', 'score', 'stand', 'phase', 'attributes']
+    return pd.read_csv(file, comment='"', header=None, names=col, sep = '\s+').drop(index=0)
```

Comment:
1. Твой код учитывает только случай, где комментарий занимает только первую строку. Однако чаще всего в файлах данного формата могут быть комментарии на много строк, их нужно учитывать. Благо в пандасе это делается супер просто
```suggestion
    return pd.read_csv(file, header=None, comment="#", names=col, sep='\s+')
```
2. `comment='"'` - это костыль, не стоит указывать символы, которые могут потенциально находится в середине строки
3. gff и bed файлы это частный случай формата tsv, значит разделителем должен быть `\t`

Reply:
Кстати, можно ещё воспользоваться функцией `pd.read_table` - это тоже самое, что `pd.read_csv(..., sep="\t")`, только этот sep туда уже вшит

Reply:
У меня этот код падает с ошибкой 
![image](https://user-images.githubusercontent.com/48160953/206883084-8c882024-3d7a-4a67-8a97-db4b43cb8caa.png)